proarc-common/src/main/resources/cz/cas/lib/proarc/common/workflow/profile/workflow.xsd

```diff
@@ -37,6 +37,7 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
     <xs:complexType name="jobDefinition">
         <xs:sequence>
             <xs:element name="worker" type="tns:workerDefinition" minOccurs="0" />
+            <xs:element name="model" type="tns:modelDefinition" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
V pořádku. Proto je dobré to mít popsané. Zvaž ještě prosím ten název atributu `@type`. Mě to evokuje něco jiného než identifikátor. `@pid` nebo `@modelId` by mohlo být výstižnější.